WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -33,6 +33,13 @@ class OrdersViewController: UIViewController {
         let storageManager = AppDelegate.shared.storageManager
         let descriptor = NSSortDescriptor(keyPath: \StorageOrder.dateCreated, ascending: false)
 
+        let futureDate = getTomorrow()
+
+        if let tomorrow = futureDate {
+            let predicate = NSPredicate(format: "dateCreated < %@", tomorrow as NSDate)
+            return ResultsController<StorageOrder>(storageManager: storageManager, sectionNameKeyPath: "normalizedAgeAsString", matching: predicate, sortedBy: [descriptor])
+        }
+
```

Comment:
Honestly it looks like the `resultsController.predicate` assignment in L159 is overwriting anything I put in here. I'm really not sure if it's good to even add the predicate here, considering that.

Reply:
@mindgraffiti I would agree with your assessment since `refreshResultsPredicate()` is called in both `viewDidLoad()` and `didChangeFilter()` — go ahead and delete this new code block.